ftw/iframeblock/browser/resources/iframeblock.js

```diff
@@ -1,9 +1,26 @@
-(function() {
+(function (factory) {
+  if (typeof define === 'function' && define.amd) {
+    require(['jquery', 'iframeResizer'], factory);
+  } else {
+    factory(window.jQuery, window.iFrameResize);
+  }
+}(function ($, iFrameResize) {
+  // We have to wait for the ready and then add the onload event listener directly on the
+  // iframe because the iframe does not propagate the onload back to document.
+  $(document).ready(() => {
+    $('iframe').on('load', function () {
+      var iframe = $(this);
 
-  "use strict";
+      if (iframe.data('autoSize') === 'True') {
+        iFrameResize({
+          log: true,
+          inPageLinks: true,
+          heightCalculationMethod: iframe.data('heightCalculationMethod'),
+          onResized: function () {scroll(0, 0);},
```

Comment:
What was the reason to switch from the documented `resizedCallback` method to this?

Reply:
The callback one is deprecated.